app/src/components/Table/KitchenRowModal.tsx

```diff
@@ -0,0 +1,99 @@
+import React from 'react';
```

Comment:
I think we should create a components folder inside the page folder and put this there.
Pages > whichever page this is > components > KitchenRowModal.tsx

Reply:
Yes I missed this! There is already a Modals folder so it can go there